docs/ddl-operation-on-carbondata.md

```diff
@@ -101,6 +101,14 @@ The following DDL operations are supported in CarbonData :
 
    - All dimensions except complex datatype columns are part of multi dimensional key(MDK). This behavior can be overridden by using TBLPROPERTIES. If the user wants to keep any column (except columns of complex datatype) in multi dimensional key then he can keep the columns either in DICTIONARY_EXCLUDE or DICTIONARY_INCLUDE.
 
+   - **Sort Columns Configuration**
+
+      It is used to specify the  multi dimensional key(MDK) columns. By default MDK is composed of all dimension columns except complex datatype column. 
+
```

Comment:
here, need give the description for "SORT_COLUMN" property:
"SORT_COLUMN"  property is for users to specify which columns belong to the MDK index.
If user don't specify "SORT_COLUMN" property, by default MDK index be built by using all dimension columns except complex datatype column.